src/openassetio-python/pyproject.toml

```diff
@@ -13,6 +13,9 @@ build-backend = "setuptools.build_meta"
 name = "openassetio"
 version = "1.0.0a6"
 requires-python = ">=3.7"
+dependencies = [
+    "importlib_metadata == 5.0"
```

Comment:
[question] My brain is a bit fried - what is the preferred way to specify this, should we pin to a specific version or greater than?

Reply:
Good point, thanks. [Downgraded to `>=3.6.0` here](https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/c1d5cbbbcf476aa05d411950d4b3bebe65c28a38).

Reply:
Great. You might have forget to modify it in the pyproject.toml file though.

Reply:
Good spot - thanks! Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/768/commits/bced6d2cd8f6d83753190425614e66cd1857fce8).